lib/proto/Raft.proto

```
@@ -0,0 +1,65 @@
+syntax = "proto3";
```

Buf (1.54.0)

Files must have a package defined.

(PACKAGE_DEFINED)